rest/admin_api.go

```diff
@@ -148,10 +148,27 @@ func (h *handler) handleReplicate() error {
 	}
 
 	params, cancel, _, err := h.readReplicationParametersFromJSON(body)
+	if err != nil {
+		return err
+	}
 
+	response, err := h.server.HTTPClient.Get(params.GetTargetDbUrl())
```

Comment:
Does this handle the case where source and/or target is a local db?

I think the params are usually just a database name in that case?

Reply:
Yeah. The GetTargetDdUrl() with auth returns something like `http://:4985/test` in that case. The http client seems to take that so yeah. It does work.